package.json

```diff
@@ -55,11 +55,6 @@
       "docs/"
     ]
   },
-  "babel": {
```

Comment:
babel wont compile unless this is gone. But it wont compile for you @paulcpederson  unless this is here?